packages/atomic/src/components/facets/atomic-date-facet/atomic-date-facet.tsx

```diff
@@ -71,25 +71,27 @@ export class AtomicDateFacet implements InitializableComponent, BaseFacetState {
   /**
    * Whether or not the index should automatically generate options for the facet
    */
-  @Prop() public generateAutomaticRanges = true;
+  @Prop() public generateAutomaticRanges?: boolean;
```

Comment:
This was the only way I could think of to allow users to manually set `generateAutomaticRanges` to true and have a combination of manual ranges and automatic ranges.

Reply:
Looking back, here's my question to everyone: is there any use case for generating automatic ranges + specifying manual ones? I vaguely remember having a discussion on that.

If there isn't a use case for that, then `generateAutomaticRanges` could be removed altogether from Atomic.

Reply:
What do you think of if we removed the `generateAutomaticRanges` prop all together? @lvu285 and I were thinking it would be simpler if developers could not directly set the value for the first version. The value would be determined by `manualRanges.length`.

Reply:
I do think there might be a use case where someone would want to specify `$25 or under` with some arbitrary fixed value and then generate the rest automatically. I'm not sure how common it would be however, so we could just wait and see if someone requests it.

Reply:
personally, I prefer to remove the option, as will be more complicate for user when they use the custom/manual range. 
I dont think there will be someone use both generating automatic ranges + specifying manual ones at the same time

Reply:
100% agree with that !

Removing the prop is the way to go.